### ISSUE TITLE
feature request: "dotnet format" mode that only formats code

### ISSUE THREAD

**Issue by fschwiet** [dotnet/format#1517]

**Comment by fschwiet** [comment]
Yeah its marked a bunch of methods in my code as Obsolete and I wish it hadn't.

**Comment by RikkiGibson** [comment]
Does `dotnet format style` do what you want?

**Comment by fschwiet** [comment]
It might be better than dotnet format whitespace if it is actually applying .editorconfig settings like csharp_style_namespace_declarations
